librz/core/cmd/cmd_api.c

```diff
@@ -1483,11 +1493,11 @@ RZ_API bool rz_cmd_get_help_json(RzCmd *cmd, const RzCmdDesc *cd, PJ *j) {
  */
 RZ_API bool rz_cmd_get_help_strbuf(RzCmd *cmd, const RzCmdDesc *cd, bool use_color, RzStrBuf *sb) {
 	rz_return_val_if_fail(cmd && cd && sb, false);
-	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary, false, MAX_RIGHT_ALIGHNMENT, use_color);
+	do_print_child_help(cmd, sb, cd, cd->name, cd->help->summary, "|", false, MAX_RIGHT_ALIGHNMENT, use_color);
 	return true;
 }
 
-RZ_API char *rz_cmd_get_help(RzCmd *cmd, RzCmdParsedArgs *args, bool use_color) {
+RZ_API char *rz_cmd_get_help(RzCmd *cmd, RzConfig *config, RzCmdParsedArgs *args, bool use_color) {
```

Comment:
Also here, you would need to add the type annotations. Also please document the function.